xla/service/gpu/autotuning/gemm_fusion_autotuner_test.cc

```diff
@@ -1148,6 +1202,9 @@ TEST_F(GemmFusionAutotunerTest, CreatesCustomKernelFusionConfigs) {
 }
 
 TEST_F(GemmFusionAutotunerTest, GeneratesConfigForUpcastGemmWithPrologue) {
+  if (isRocm()) {
+    GTEST_SKIP() << "Not supported on ROCm.";
+  }
```

Comment:
may I ask why `GeneratesConfigForUpcastGemmWithPrologue)` and `GeneratesConfigForUpcastGemmWithPrologueAndEpilogue` are not supported?